heartbeat/aws-vpc-move-ip

```diff
@@ -159,10 +159,12 @@ Enable enhanced monitoring using AWS API calls to check route table entry
 
 <parameter name="lookup_type" required="0">
 <longdesc lang="en">
-Name of resource type to lookup in route table. In shared subnet setups use NetworkInterfaceId.
+Name of resource type to lookup in route table.
+"InstanceId"         : (default)
+"NetworkInterfaceId" : usefull in shared vpc setups.
```

Comment:
Looks good. Minor things:
s/usefull/useful/
s/vpc/VPC/ (no big deal but if we're changing 'usefull', might as well change this too)

I would say:
```
"InstanceId"         : Default value, EC2 instance ID.
"NetworkInterfaceId" : ENI ID (useful in shared VPC setups).
```

The rest of the PR now looks good to me, in terms of code conventions and whatnot :) I don't have the cycles to research and test the overall approach, so I'll trust that it all makes sense, and see if Oyvind has anything to add in that regard.